spokestack/extensions/webrtc/vad.pyx

```diff
@@ -0,0 +1,41 @@
+cimport cvad
+cimport numpy as np
+
+
+np.import_array()
+
+cdef class WebRtcVad:
+    cdef cvad.VadInst*_vad
+    cdef int _sample_rate
+    cdef int _mode
+
+    def __dealloc__(self):
+        cvad.WebRtcVad_Free(self._vad)
+
+    def __init__(self, sample_rate=16000, mode=0):
+        self._sample_rate = sample_rate
+        self._mode = mode
+
+        result = cvad.WebRtcVad_Create(&self._vad)
+        if result == 0:
```

Comment:
we are silently ignoring failure in all of these `if result` checks
failures in these checks should result in exceptions